src/main/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocator.groovy

```diff
@@ -127,7 +130,8 @@ class RegistryAuthLocator {
             }
 
         } catch(Exception ex) {
-            logger.error('Failure when attempting to lookup auth config ' +
+            logger.warn('Failure when attempting to lookup auth config ' +
```

Comment:
For now, I'd not want to change the log level for anything that might go wrong. It probably indicates a real issue. If the parsing of the JSON file doesn't work properly then that's a bug.

So I'd still want to see an error for the following cases:
1. The JSON cannot be parsed.
2. The auth config cannot be determined.
3. The credentials cannot be resolved.

IMHO those are all valid use cases. So if you have a config file then I'd expect it to work.